charts/kube2iam/README.md

```diff
@@ -0,0 +1,100 @@
+
+# kube2iam
+
+Installs [kube2iam](https://github.com/jtblin/kube2iam) to provide IAM credentials to pods based on annotations.
+
+## TL;DR;
+
+```console
+$ helm install stable/kube2iam
+```
+
+## Introduction
+
+This chart bootstraps a [kube2iam](https://github.com/jtblin/kube2iam) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+  - Kubernetes 1.4+ with Beta APIs enabled
+
+## Add repos & install the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+$ helm repo add kube2iam https://buildkite.github.io/charts/
```

Comment:
Pretty sure this URL is not correct. That repo is for Buildkite, not this chart. Was Github Pages set up for this repo? Once it is, I believe this URL should be `https://<pages_name>.github.io/charts` (or something along those lines).